.husky/commit-msg

```diff
@@ -2,4 +2,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn commit-msg
+npx --no-install commitlint --edit "$1"
```

Comment:
Why do we need to run this now with `npx`?

Reply:
I am honestly not really sure, but our previous setup was failing. And they do exactly the following in the official docs https://github.com/conventional-changelog/commitlint So it's also nice for us as it removes a hook script from `package.json`.